store/store_test.go

```diff
@@ -30,7 +32,9 @@ func initStore(t *testing.T) (*store.Store, error) {
 }
 
 func TestUpdate(t *testing.T) {
-	s, err := initStore(t)
+	tempDir, err := ioutil.TempDir("", "sth")
```

Comment:
Note that the `testing` package provides its own `TempDir` function: https://pkg.go.dev/testing@go1.16.5#T.TempDir